modules/web-demo/src/components/Coins/coinFactory.ts

```diff
@@ -0,0 +1,196 @@
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { BaseCoin } from '@bitgo/sdk-core';
+
+class CoinFactory {
```

Comment:
should we create a CoinFactory in `sdk-api`?
`sdk-api` already depends on `sdk-core`

Reply:
No, `sdk-api` can't import coin specific modules - it must remain decoupled. This class is purely for demonstration purposes and has no bearing on the published code.